packages/table/src/index.ts

```diff
@@ -1,6 +1,6 @@
-export { default as Cell } from './Cell';
-export { default as HeaderRow } from './HeaderRow';
-export { default as Row } from './Row';
-export { default as Table } from './Table';
-export { DataType } from './TableContext';
-export { default as TableHeader } from './TableHeader';
+import Table, { TableProps } from './Table';
+
+export * from '@tanstack/react-table';
+export { Table, type TableProps };
+
+export default Table;
```

Comment:
I may have missed it, but are you exporting all the necessary components?

Reply:
^